vars/runPactTest.groovy

```diff
@@ -2,16 +2,17 @@
 def call( String providerProjectName,
           String consumerTag) {
 
-    def providerSha = gitCommit()
     gitClone(gitUriFromProjectName(providerProjectName))
+    def providerSha = sh(script: "cd ${providerProjectName} && git rev-parse HEAD", returnStdout: true).trim()
```

Comment:
not sure if there's a better way to do this

Reply:
Let's make it work, we can refactor